src/api/issues.rs

```diff
@@ -775,3 +775,346 @@ impl<'octo, 'r> ListIssueCommentsBuilder<'octo, 'r> {
         self.handler.crab.get(route, Some(&self)).await
     }
 }
+
+impl<'octo> IssueHandler<'octo> {
+    /// Lists reactions for an issue.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let reactions = octocrab::instance()
+    ///     .issues("owner", "repo")
+    ///     .list_reactions(1)
+    ///     .per_page(100)
+    ///     .page(2u32)
+    ///     .send()
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn list_reactions(&self, issue_number: u64) -> ListReactionsBuilder<'_, '_> {
+        ListReactionsBuilder::new(self, issue_number)
+    }
+
+    /// Lists reactions for an issue comment.
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    /// let reactions = octocrab::instance()
+    ///     .issues("owner", "repo")
+    ///     .list_comment_reactions(1)
+    ///     .per_page(100)
+    ///     .page(2u32)
+    ///     .send()
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn list_comment_reactions(&self, comment_id: u64) -> ListCommentReactionsBuilder<'_, '_> {
+        ListCommentReactionsBuilder::new(self, comment_id)
+    }
+}
+
+#[derive(serde::Serialize)]
+pub struct ListReactionsBuilder<'octo, 'r> {
+    #[serde(skip)]
+    handler: &'r IssueHandler<'octo>,
+    issue_number: u64,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    per_page: Option<u8>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    page: Option<u32>,
+}
+
+impl<'octo, 'r> ListReactionsBuilder<'octo, 'r> {
+    pub(crate) fn new(handler: &'r IssueHandler<'octo>, issue_number: u64) -> Self {
+        Self {
+            handler,
+            issue_number,
+            per_page: None,
+            page: None,
+        }
+    }
+
+    /// Results per page (max 100).
+    pub fn per_page(mut self, per_page: impl Into<u8>) -> Self {
+        self.per_page = Some(per_page.into());
+        self
+    }
+
+    /// Page number of the results to fetch.
+    pub fn page(mut self, page: impl Into<u32>) -> Self {
+        self.page = Some(page.into());
+        self
+    }
+
+    /// Send the actual request.
+    pub async fn send(self) -> Result<crate::Page<models::reactions::Reaction>> {
+        let route = format!(
+            "repos/{owner}/{repo}/issues/{issue}/reactions",
+            owner = self.handler.owner,
+            repo = self.handler.repo,
+            issue = self.issue_number,
+        );
+
+        self.handler.crab.get(route, Some(&self)).await
+    }
+}
+
+#[derive(serde::Serialize)]
+pub struct ListCommentReactionsBuilder<'octo, 'r> {
+    #[serde(skip)]
+    handler: &'r IssueHandler<'octo>,
+    comment_id: u64,
```

Comment:
Shouldn't the type be `CommentId`? Otherwise a user might have to use extra code to change the type in their code?

Reply:
I followed the way the current code does it, example https://github.com/XAMPPRocky/octocrab/blob/master/src/api/issues.rs#L658

Reply:
Followup PR always welcome 🙂

Reply:
Thanks, done in https://github.com/XAMPPRocky/octocrab/pull/274